app/internal/out_of_office.py

```diff
@@ -0,0 +1,93 @@
+from datetime import datetime
+from sqlalchemy.orm import Session
+
+from app.database.models import User, OutOfOffice
+
+
+def get_who_is_out_of_office(session: Session,
+                             event_start_date,
+                             invited_emails):
+    """
+    This func check who is out of office
+    :param session: db session
+    :param event_start_date: event start date
+    :param invited_emails: invited users
+    :return: all users that cant be at the meeting
+    """
+    out_of_office_users = session.query(User.username,
+                                        User.email).join(OutOfOffice).filter(
+        User.email.in_(invited_emails)).filter(OutOfOffice.start_date
+                                               <= event_start_date,
+                                               OutOfOffice.end_date
+                                               >= event_start_date).filter(
+        OutOfOffice.status == 'On').all()
+    return out_of_office_users
+
+
+def insert_new_out_of_office(out_of_office_data,
+                             user,
+                             session):
+    out = get_out_of_office_template(1,
+                                     user_id=user.id,
+                                     start_date=datetime.
+                                     strptime(
+                                         out_of_office_data['start_date']
+                                         + ' ' +
+                                         out_of_office_data['start_time'],
+                                         '%Y-%m-%d %H:%M'),
+                                     end_date=datetime.strptime(
+                                         out_of_office_data['end_date']
+                                         + ' ' +
+                                         out_of_office_data['end_time'],
+                                         '%Y-%m-%d %H:%M'),
+                                     status='On')
+    session.add(out)
+
+
+def get_out_of_office_template(out_of_office_id,
+                               user_id,
+                               start_date=None,
+                               end_date=None,
+                               status='Off'):
+    return OutOfOffice(
+        id=out_of_office_id,
+        user_id=user_id,
+        start_date=start_date,
+        end_date=end_date,
+        status=status
+    )
+
+
+def update_out_of_office(out_of_office_data_from_req,
+                         out_of_office_data_from_db):
+    activate_out_of_office = '1'
```

Comment:
please add TODO

Reply:
Explain, please :)

Reply:
Maybe could you please explain this line? why you set always '1'?